NuGet.Config

```diff
@@ -4,6 +4,7 @@
     <clear />
     <add key="nuget.org" value="https://api.nuget.org/v3/index.json" />
     <add key="dotnet-tools" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-tools/nuget/v3/index.json" />
+    <add key="azure-sdk-tools" value="https://pkgs.dev.azure.com/azure-sdk/public/_packaging/azure-sdk-tools/nuget/v3/index.json" />
```

Comment:
I'd like to avoid adding another feed to this nuget.config. I want to eventually remove all but one if possible.